packages/atom-editor-deb/atom-editor-deb.pacscript

```diff
@@ -1,7 +1,8 @@
 name="atom-editor-deb"
 gives="atom"
-version="1.59.0"
+repology="atom"
+version="1.60.0"
```

Comment:
So, `atom` has 2 newest versions in repology,`1.60` (from nix) and `1.60.0` (from others)

https://repology.org/project/atom/versions

So out of the `1.60.0` ones (as this pacscript uses the `x.y.z` version format), which repository do you want to track?

Reply:
I'll be tracking `1.60.0` version as it is most common.

Reply:
Okay, I've updated Pacup to use the most common version out of the filtrate by default
https://github.com/pacstall/pacup/commit/fec2373f1f1585df0bb94223c75673d2d30ee534